### PR TITLE
Deployment tooling update - bin/eb-release

### PR DESCRIPTION
**Pull request by indigobravo**
The bin/eb-release script has been updated to support multi-region
deployments

### CODE REVIEW

File: bin/eb-release [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 usage () {
-    echo "Usage: $(basename "$0") <APP> <APP_DOCKER_TAG> <APP_ENV>"
+    echo "Usage: $(basename "$0") <APP> <APP_DOCKER_TAG> <APP_ENV> <REGION>"
```
Comment (indigobravo) [inline bin/eb-release:4]: Updated help message

File: bin/eb-release [hunk 2]
```diff
@@ -23,7 +24,7 @@ status () {
     echo "--->" "$@" >&2
 }
 
-if [ "$#" -ne 3 ]; then
+if [ "$#" -ne 4 ]; then
```
Comment (indigobravo) [inline bin/eb-release:27]: Set mandatory parameters to 4 now we are passing in `REGION`

File: bin/eb-release [hunk 3]
```diff
@@ -35,6 +36,7 @@ PATH="$(dirname "$0"):${PATH}"
 APP=$1
 APP_DOCKER_TAG=$2
 APP_ENV=$3
+REGION=$4
```
Comment (indigobravo) [inline bin/eb-release:39]: Set `REGION` to `$4`

File: bin/eb-release [hunk 4]
```diff
@@ -79,7 +81,7 @@ trap 'rm -r "$SRCBUNDLEDIR"' EXIT
 
 status "fetching storage location"
 
-EB_BUCKET=$(aws elasticbeanstalk create-storage-location --query S3Bucket --output text)
+EB_BUCKET=$(aws elasticbeanstalk create-storage-location --query S3Bucket --output text --region "$REGION")
```
Comment (indigobravo) [inline bin/eb-release:84]: Added `--region` to `aws` command.

File: bin/eb-release [hunk 5]
```diff
@@ -103,7 +105,8 @@ aws elasticbeanstalk create-application-version \
     --process \
     --auto-create-application \
     --query 'ApplicationVersion.VersionLabel' \
-    --output text
+    --output text \
+    --region "$REGION"
```
Comment (indigobravo) [inline bin/eb-release:109]: Added `--region` to `aws` command

File: bin/eb-release [hunk 6]
```diff
@@ -112,7 +115,8 @@ while true; do
                --application-name "$APP" \
                --version-label "$VERSION_LABEL" \
                --query 'ApplicationVersions[0].Status' \
-               --output text)
+               --output text \
+               --region "$REGION")
```
Comment (indigobravo) [inline bin/eb-release:119]: Added `--region` to `aws` command

**Review by robertknight — APPROVED** [review]

**Pull request merged**